mlflow/models/resources.py

```diff
@@ -85,21 +103,17 @@ class DatabricksUCConnection(DatabricksResource):
     Args:
         connection_name (str): The name of the databricks UC connection
         used to create the tool which was used to build the model.
+        on_behalf_of_user (Optional[bool]): If True, the resource is accessed with
+        with the permission of the invoker of the model in the serving endpoint. If set to
+        None or False, the resources is accesssed with definers rights
```

Comment:
```suggestion
        None or False, the resources is accesssed with definers rights (the owner of the model)
```

nit: 'definer' might not be familiar term for ppl who came from DS/ML background.